src/components/CountryPicker/index.js

```diff
@@ -48,6 +52,15 @@ function CountryPicker({value, errorText, onInputChange, forwardedRef}) {
     const hidePickerModal = () => {
         setIsPickerVisible(false);
     };
+    const closeModal = () => {
+        hidePickerModal();
+        // if its macOS app or desktop browser, dismiss the modal when you click on the backdrop
+        if (getPlatform() === CONST.PLATFORM.DESKTOP || getPlatform() === CONST.PLATFORM.WEB) {
+            if (!Browser.isMobile()) {
+                Navigation.dismissModal();
+            }
+        }
```

Comment:
I think this way will be more clear and we can avoid this platform check if its mobile device.

```
hidePickerModal();
        
if (Browser.isMobile()) {
    return;
}
        
if (getPlatform() === CONST.PLATFORM.DESKTOP || getPlatform() === CONST.PLATFORM.WEB) {
    Navigation.dismissModal();
}
```

Reply:
I agree. `isMobile()` will return `true` on mobile browsers and `false` on other platforms. To further filter out native devices, we can do 
```
if (getPlatform() === CONST.PLATFORM.DESKTOP || getPlatform() === CONST.PLATFORM.WEB) {
    Navigation.dismissModal();
} 
```

Reply:
I have pushed the update